superdesk/utils.py

```diff
@@ -154,3 +155,11 @@ def compare_preferences(original, updates):
     removed = original_keys - updates_keys
     modified = {o: (original[o], updates[o]) for o in intersect_keys if original[o] != updates[o]}
     return added, removed, modified
+
+
+def sha(text):
```

Comment:
Aha, I see, it does that kind of make sense.
(unless the whole SHA family gets broken :stuck_out_tongue_winking_eye: )